Bugzilla/DaemonControl.pm

```diff
@@ -0,0 +1,317 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::DaemonControl;
+use 5.10.1;
+use strict;
+use warnings;
+
+use Cwd qw(realpath);
+use English qw(-no_match_vars $PROGRAM_NAME);
+use File::Spec::Functions qw(catfile);
+use Future::Utils qw(repeat try_repeat);
+use Future;
+use IO::Async::Loop;
+use IO::Async::Process;
+use IO::Async::Protocol::LineStream;
+use IO::Async::Signal;
+use LWP::Simple qw(get);
+use POSIX qw(setsid WEXITSTATUS);
+use WWW::Selenium::Util qw(server_is_running);
```

Comment:
is it possible to have different sets of packages for prod vs test?  i don't think selenium should be part of the production deployment requirements; instead it can be loaded at runtime during test.